src/pharmpy/results.py

```diff
@@ -5,8 +5,9 @@
 from collections.abc import MutableSequence
 from pathlib import Path
 
-from pharmpy.deps import altair as alt
-from pharmpy.deps import pandas as pd
+import altair as alt
+import pandas as pd
```

Comment:
Is there a need to change these imports?

Reply:
No, sorry, I think these were accidentally added when I merged the new imports, will change!

Reply:
OK. I see it is resolved now. That was supposed to be caught by `tox -e py38-lint` but the regexes were off. I fixed it now so this does not happen again in the future.